Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes the following components:

* [Jamulus/Copyright notices](https://hosted.weblate.org/projects/jamulus/copyright-notices/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Unregistered-Servers](https://hosted.weblate.org/projects/jamulus/unregistered-servers/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/jamulus/1-index/horizontal-auto.svg)

ann0see commented:
Somehow the iOS changes didn't show up on Weblate. I needed to reload the po files. Hopefully nothing is lost.

ann0see commented:
Screenshots for sv-se are missing

ignotus666 commented:
No, they're there. It was because of entering the lang code as `sv_SE` instead of `sv-SE`. The build is going to keep failing until all the screenshots that have had their underscores changed to dashes are corrected - now it's because of `pt_PT.`

ignotus666 commented on _translator-files/po/sv-SE/1-index.po:
```diff
@@ -81,33 +98,35 @@ msgid ""
 #, no-wrap
 msgid "Take a look at our [contribution guidelines](wiki/Contribution) to find out how. Everybody is welcome!"
-msgstr ""
+msgstr "Ta en titt på våra [contribution guidelines](wiki/Contribution) för att ta reda på hur du kan bidra. Alla är välkomna!"
```
Content in square brackets needs translating.

pljones replied:
`id="bidra-till-jamulusprojektet"`

ignotus666 commented on _translator-files/po/sv-SE/Client-Troubleshooting.po:
```diff
@@ -70,276 +71,278 @@ msgid ""
 #, markdown-text, no-wrap
 msgid "**Windows users (ASIO4ALL)**: If you’re using the ASIO4ALL driver have a look at the [ASIO4ALL setup section](Installation-for-Windows#setting-up-asio4all)\n"
-msgstr ""
+msgstr "**Windows-användare (ASIO4ALL)**: Om du använder ASIO4ALL-drivrutinen ta en titt på [ASIO4ALL-installationsavsnittet](Installation-for-Windows#setting-up-asio4all)\n"
```
Text in link after # should be translated - with text that matches the corresponding paragraph title.

ann0see replied:
@genesisproject2020 could you please fix those issues - either on weblate or in a separate PR to this repo?

ignotus666 commented on _translator-files/po/sv-SE/1-index.po:
```diff
@@ -81,33 +98,35 @@ msgid ""
 "You can also ask on the [forums](https://github.com/jamulussoftware/jamulus/discussions).\n"
 msgstr ""
+"Kolla in [dokumentationen](wiki/Getting-Started) och överväg att läsa avsnittet om [troubleshooting section](wiki/Client-Troubleshooting)! \n"
+"Du kan också ställa frågor på [forums](https://github.com/jamulussoftware/jamulus/discussions).\n"
```
Not sure if "forums" is correct in Swedish.

pljones replied:
Text in square brackets needs translating (`[troubleshooting section]`), too.

ignotus666 commented on _translator-files/po/sv-SE/Client-Troubleshooting.po:
```diff
@@ -70,276 +71,278 @@ msgid ""
 #, markdown-text
 msgid "In some cases, it may be your ISP that is blocking your use of Jamulus. See the note on the [server troubleshooting page](Server-Troubleshooting#nobody-can-connect-to-my-server---but-i-can-connect-locally) about \"Nobody can connect to my Server\""
-msgstr ""
+msgstr "I vissa fall kan det vara din internetleverantör som blockerar din användning av Jamulus. Se anteckningen på [serverfelsökningssidan](Server-felsökning#ingen-kan-ansluta-till-min-server---men-jag-kan-ansluta-lokalt) om \"Ingen kan ansluta till min server\""
```
"Server-Troubleshooting" in link shouldn't be translated.

ann0see replied:
Crucial to get right.

ignotus666 commented on _translator-files/po/sv-SE/Command-Line-Options.po:
```diff
@@ -1,82 +1,83 @@
 #, markdown-text
 msgid "See [Running a Server](Running-a-Server#server-mode-related-options)"
-msgstr ""
+msgstr "Se [Köra en server](Running-a-Server#server-mode-related-options)"
```
Translate text after #.

pljones replied:
`serverrelaterade-alternativ`

ignotus666 commented on _translator-files/po/sv-SE/Directories.po:
```diff
@@ -1,112 +1,113 @@
 #, markdown-text
 msgid "{% include breadcrumb.html root=\"More\" branch1=\"Server Administration\" branch1-url=\"Running-a-Server\" %}"
-msgstr ""
+msgstr "{% include breadcrumb.html root=\"Mer\" branch1=\"Serveradministration\" branch1-url=\"Köra-en-server\" %}"
```
" branch1-url=\"Running-a-Server\" shouldn't be translated.

ann0see replied:
This one is crucial to get right.

ignotus666 commented on _translator-files/po/sv-SE/Directories.po:
```diff
@@ -1,112 +1,113 @@
 #, markdown-text
 msgid "This is a specialised Jamulus Server configuration, as described in [Server Modes](Running-a-Server#server-modes)."
-msgstr ""
+msgstr "Detta är en specialiserad Jamulus Server-konfiguration, som beskrivs i [Serverlägen](Running-a-Server#server-modes)."
```
Translate text after #.

pljones replied:
`serverlägen`

ignotus666 commented on _translator-files/po/sv-SE/FAQ.po:
```diff
@@ -70,165 +71,167 @@ msgid ""
 #, markdown-text
 msgid "Bear in mind also that all participants should follow [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) which will also determine whether you can play in time properly."
-msgstr ""
+msgstr "Tänk också på att alla deltagare bör följa [Den gyllene regeln](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together), vilket också kommer att påverka om ni kan spela i takt på rätt sätt."
```
Translate text after #.

pljones replied:
`ni-låter-alla-okej-men-det-är-svårt-att-hålla-ihop`

ignotus666 commented on _translator-files/po/sv-SE/FAQ.po:
```diff
@@ -70,165 +71,167 @@ msgid ""
 #, markdown-text
 msgid "Note also that Servers do not have to be registered on a Directory in order for Jamulus Clients to connect to them. Server operators can just give out the address of their Servers to those they want to play with, and they can then connect directly. Have a look at [this overview for more information](/wiki/Running-a-Server#server-modes)."
-msgstr ""
+msgstr "Observera också att servrar inte behöver vara registrerade i en katalog för att Jamulus-klienter ska kunna ansluta till dem. Serveroperatörer kan helt enkelt ge ut adressen till sina servrar till de vill spela med, och de kan då ansluta direkt. Ta en titt på [denna översikt för mer information](/wiki/Running-a-Server#server-modes)."
```
Translate text after #.

pljones replied:
`serverlägen`

ignotus666 commented on _translator-files/po/sv-SE/FAQ.po:
```diff
@@ -70,165 +71,167 @@ msgid ""
 #, markdown-text
 msgid "Whether or not you will be able to play in time with other musicians depends mainly on how much latency (delay) you have in your sound signal and whether you are all following [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together). An overall delay much bigger than 50 ms usually makes it too hard to play in time unless you train yourself to do s ...
-msgstr ""
+msgstr "Om du kan spela i takt med andra musiker beror främst på hur mycket latens (fördröjning) du har i ditt ljudsignal och om ni alla följer [Den gyllene regeln](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together). En total fördröjning som är mycket större än 50 ms gör det vanligtvis för svårt att spela i takt, om du inte tränar dig själv att göra det. Vissa musik ...
```
Translate text after # in link.

pljones replied:
`ni-låter-alla-okej-men-det-är-svårt-att-hålla-ihop`

ignotus666 commented on _translator-files/po/sv-SE/FAQ.po:
```diff
@@ -70,165 +71,167 @@ msgid ""
 #, markdown-text
 msgid "For the same reason as you need to minimise delay in your signal in order to play in time, you need to make sure you are playing to your own sound that other musicians are hearing. More information on this, and a way of testing your setup to make sure you are obeying this “Golden Rule” [can be found here](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together)."
-msgstr ""
+msgstr "Av samma anledning som du behöver minimera fördröjning i din signal för att spela i takt, måste du se till att du spelar till ditt eget ljud som de andra musikerna hör. Mer information om detta och ett sätt att testa din setup för att säkerställa att du följer denna \"gyllene regel\" [kan du hitta här](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together)."
```
Translate text after # in link.

pljones replied:
`ni-låter-alla-okej-men-det-är-svårt-att-hålla-ihop`

ignotus666 commented on _translator-files/po/sv-SE/Getting-Started.po:
```diff
@@ -1,91 +1,92 @@
 #, markdown-text
 msgid "**A wired internet connection** (and turn wi-fi off. See the [FAQ](/wiki/FAQ#why-shouldnt-i-use-wireless-equipment))"
-msgstr ""
+msgstr "**En trådbunden internetanslutning** (och stäng av Wi-Fi. Se [FAQ](/wiki/FAQ#why-shouldnt-i-use-wireless-equipment))"
```
Translate text after # in link.

pljones replied:
`varför-bör-jag-inte-använda-trådlös-utrustning`

ignotus666 commented on _translator-files/po/sv-SE/Getting-Started.po:
```diff
@@ -1,91 +1,92 @@
 #, markdown-text
 msgid "**Wired headphones** (not Bluetooth or speakers - see the [FAQ](/wiki/FAQ#why-shouldnt-i-use-wireless-equipment))"
-msgstr ""
+msgstr "**Trådbundna hörlurar** (inte Bluetooth eller högtalare – se [FAQ](/wiki/FAQ#why-shouldnt-i-use-wireless-equipment))"
```
Translate text after # in link.

pljones replied:
`varför-bör-jag-inte-använda-trådlös-utrustning`

ignotus666 commented on _translator-files/po/sv-SE/Getting-Started.po:
```diff
@@ -1,91 +1,92 @@
 #, markdown-text
 msgid "**A reasonable audio device, sound card and/or mic** ([see this list](/kb/2021/01/05/Jamulus-Sound-Devices.html) for examples)"
-msgstr ""
+msgstr "**En rimlig ljudenhet, ljudkort och/eller mikrofon** ([see this list](/kb/2021/01/05/Jamulus-Sound-Devices.html) för exempel)"
```
Text in square brackets should be translated.

ignotus666 commented on _translator-files/po/sv-SE/Getting-Started.po:
```diff
@@ -160,36 +171,40 @@ msgid ""
 #, markdown-text
 msgid "Before you connect to a Server, you should **not** be able to hear yourself.  Listening to your sound from Jamulus (and not from yourself) is called “The Golden Rule” and enables you to play in time with others ([see the FAQ](/wiki/FAQ#why-should-i-not-listen-to-my-own-signal))."
-msgstr ""
+msgstr "Innan du ansluter till en server bör du **inte** kunna höra dig själv. Att lyssna på ditt ljud från Jamulus (och inte direkt från dig själv) kallas för \"Den gyllene regeln\" och gör det möjligt för dig att spela i takt med andra ([se FAQ](/wiki/FAQ#why-should-i-not-listen-to-my-own-signal))."
```
Translate text after # in link.

pljones replied:
`varför-bör-jag-inte-lyssna-på-mitt-eget-ljud`

ignotus666 commented on _translator-files/po/sv-SE/Getting-Started.po:
```diff
@@ -200,30 +215,34 @@ msgid ""
 #, markdown-text
 msgid "Once connected to a Server, make sure you can hear yourself OK and fix any input volume or other problems. You can have a look at the [troubleshooting page](/wiki/Client-Troubleshooting) for common problems. And of course, check that you are also able to obey [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together)."
-msgstr ""
+msgstr "När du är ansluten till en server, se till att du kan höra dig själv ordentligt och åtgärda eventuella problem med ingångsvolymen eller andra problem. Du kan titta på [troubleshooting page](/wiki/Client-Troubleshooting) för vanliga problem. Och självklart, kontrollera att du också kan följa [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) ...
```
Text in square brackets should be translated and translate text after # in link.

pljones replied:
`[Den gyllene regeln]` and `ni-låter-alla-okej-men-det-är-svårt-att-hålla-ihop`

ignotus666 commented on _translator-files/po/sv-SE/Getting-Started.po:
```diff
@@ -234,33 +253,37 @@ msgid ""
 #, markdown-text
 msgid "More information about using Jamulus can be found in the [Software Manual](/wiki/Software-Manual)."
-msgstr ""
+msgstr "Mer information om hur du använder Jamulus hittar du i [Software Manual](/wiki/Software-Manual)."
```
Translate text in square brackets.

pljones replied:
We should change the English here, really... the page title is "Jamulus User Manual" (although "Jamulus" is probably redundant there), which more accurately reflects its content (as opposed to the server manual), which would give "användarmanual" here for the translation.

(Issue raised.)

ignotus666 commented on _translator-files/po/sv-SE/Include-Client-Commands.po:
```diff
@@ -1,52 +1,52 @@
 #, markdown-text
 msgid "`--ctrlmidich` MIDI controller channel to listen on, control number offset and consecutive CC numbers (channels) and Mute Myself CC number. Format: `channel[;f*][;p*][;s*][;m*][;o]` See [Tips & Tricks](Tips-Tricks-More#using-ctrlmidich-for-midi-controllers)"
-msgstr ""
+msgstr "`--ctrlmidich` MIDI-kontrollerkanal att lyssna på, kontrollnummer-offset och efterföljande CC-nummer (kanaler) och Mute Myself CC-nummer. Format: `channel[;f*][;p*][;s*][;m*][;o]` Se [Råd & dåd](Tips-Tricks-More#using-ctrlmidich-for-midi-controllers)"
```
Translate text after # in link.

pljones replied:
`använder-ctrlmidich-för-midi-kontroller`

ignotus666 commented on _translator-files/po/sv-SE/Installation-for-Android.po:
```diff
@@ -1,131 +1,132 @@
 #, markdown-text, no-wrap
 msgid "Installation for Android"
-msgstr ""
+msgstr "Installation för Android"
```
Should be translated.

ignotus666 commented on _translator-files/po/sv-SE/Installation-for-Android.po:
```diff
@@ -1,131 +1,132 @@
 #, markdown-text
 msgid "Make sure you've already read the [Getting Started](Getting-Started) page."
-msgstr ""
+msgstr "Se till att du redan har läst sidan [Getting Started](Getting-Started)."
```
Translate text in square brackets.

pljones replied:
Possibly "komma igång" as it's in the page title:
> Setup - getting started with Jamulus

becomes
> Installation - komma igång med Jamulus

The English table of contents says "Setting Up", which isn't in the page title or the page name..!

ignotus666 commented on _translator-files/po/sv-SE/Installation-for-Linux.po:
```diff
@@ -1,267 +1,268 @@
 #, markdown-text, no-wrap
 msgid "Installation for Linux"
-msgstr ""
+msgstr "Installation för Linux"
```
Should be translated.

ignotus666 commented on _translator-files/po/sv-SE/Installation-for-Linux.po:
```diff
@@ -1,267 +1,268 @@
 #, markdown-text, no-wrap
 msgid "**Upgrading?** You may want to [back up your configuration](Software-Manual#backing-up-jamulus) first.\n"
-msgstr ""
+msgstr "**Uppgraderar du?** Du kanske vill [säkerhetskopiera din konfiguration](Software-Manual#backing-up-jamulus) först.\n"
```
Translate text after # in link.

pljones replied:
`säkerhetskopiera-jamulus`

ignotus666 commented on _translator-files/po/sv-SE/Installation-for-Linux.po:
```diff
@@ -1,267 +1,268 @@
 #, markdown-text
 msgid "The Jamulus Client needs to connect to a running [JACK](https://jackaudio.org/) server in order to start. You need to find out which is the low-latency sound server for your distribution.  - If your distribution uses [JACK](https://jackaudio.org/), see how to [configure JACK with QjackCtl](Installation-for-Linux#configure-jack-with-qjackctl).  - If your distribution uses PipeWire, see how  ...
-msgstr ""
+msgstr "Jamulus-klienten behöver ansluta till en körande [JACK](https://jackaudio.org/) server för att starta. Du behöver ta reda på vilken ljudserver med låg latens som används i din distribution..  - Om din distribution använder [JACK](https://jackaudio.org/), se hur du [konfigurerar JACK med QjackCtl](Installation-for-Linux#configure-jack-with-qjackctl)..  - Om din distribution använder PipeWi ...
```
Translate text after # in link.

ignotus666 commented on _translator-files/po/sv-SE/Installation-for-Linux.po:
```diff
@@ -1,267 +1,268 @@
 #, markdown-text
 msgid "Restart JACK for the new settings to take effect and continue from the [Start Jamulus](Installation-for-Linux#start-jamulus) section below."
-msgstr ""
+msgstr "Starta om JACK för att de nya inställningarna ska träda i kraft och fortsätt från avsnittet [Starta Jamulus](Installation-for-Linux#start-jamulus) nedan."
```
Translate text after # in link.

ignotus666 commented on _translator-files/po/sv-SE/Installation-for-Linux.po:
```diff
@@ -270,45 +271,47 @@ msgid ""
 #, markdown-text
 msgid "After this, continue from the [Start Jamulus](Installation-for-Linux#start-jamulus) section below."
-msgstr ""
+msgstr "Efter detta, fortsätt från avsnittet [Starta Jamulus](Installation-for-Linux#start-jamulus) nedan."
```
Translate text after # in link.

ignotus666 commented on _translator-files/po/sv-SE/Installation-for-Macintosh.po:
```diff
@@ -1,68 +1,69 @@
 #, markdown-text, no-wrap
 msgid "Installation for macOS"
-msgstr ""
+msgstr "Installation för macOS"
```
Should be translated.

ignotus666 commented on _translator-files/po/sv-SE/Installation-for-Macintosh.po:
```diff
@@ -1,68 +1,69 @@
 #, markdown-text
 msgid "Upgrading? You may want to [back up your configuration](Software-Manual#backing-up-jamulus) first."
-msgstr ""
+msgstr "Uppgradera? Du kanske vill [back up your configuration](Software-Manual#backing-up-jamulus) först."
```
Translate text after # in link.

ignotus666 commented on _translator-files/po/sv-SE/Installation-for-Windows.po:
```diff
@@ -1,74 +1,75 @@
 #, markdown-text
 msgid "Upgrading? You may want to [back up your configuration](Software-Manual#backing-up-jamulus) first."
-msgstr ""
+msgstr "Uppgradera? Du kanske vill [back up your configuration](Software-Manual#backing-up-jamulus) först."
```
Translate text after # in link.

ignotus666 commented on _translator-files/po/sv-SE/Installation-for-Windows.po:
```diff
@@ -80,144 +81,149 @@ msgid ""
 #, markdown-text, no-wrap
 msgid "**Tip:** Set up your sound card while you're [connected to a Server](Getting-Started#connecting-to-a-server-and-testing-your-sound) to hear your instrument or voice and check if everything is correctly set up; but first read on.\n"
-msgstr ""
+msgstr "**Tips:** Ställ in ditt ljudkort medan du är [ansluten till en server](Getting-Started#connecting-to-a-server-and-testing-your-sound) för att höra ditt instrument eller din röst och kontrollera om allt är korrekt inställt; men läs vidare först.\n"
```
Translate text after # in link.

ignotus666 commented on _translator-files/po/sv-SE/Installation-for-iOS.po:
```diff
@@ -1,184 +1,185 @@
 #, markdown-text
 msgid "Make sure you've already read the [Getting Started](Getting-Started) page."
-msgstr ""
+msgstr "Se till att du redan har läst sidan [Getting Started](Getting-Started)."
```
Translate text in square brackets.

ignotus666 commented on _translator-files/po/sv-SE/Running-a-Server.po:
```diff
@@ -70,126 +71,128 @@ msgid ""
 #, markdown-text
 msgid "Using a public Server might introduce you to strangers. If you want an undisturbed session, you can use the soloing technique described on the [Tips and Tricks page](Tips-Tricks-More#have-an-undisturbed-session-on-any-server). You won't hear strangers who connect to the Server, but they can hear you if they wish."
-msgstr ""
+msgstr "Att använda en offentlig server kan leda till att du kommer i kontakt med främlingar. Om du vill ha en ostörd session kan du använda solo-tekniken som beskrivs på [Tips och tricks-sidan](Tips-Tricks-More#have-an-undisturbed-session-on-any-server). Du kommer inte att höra främlingar som ansluter till servern, men de kan höra dig om de så önskar."
```
Translate text after # in link.

ignotus666 commented on _translator-files/po/sv-SE/Running-a-Server.po:
```diff
@@ -70,126 +71,128 @@ msgid ""
 #, markdown-text
 msgid "Most people run Jamulus on a 3rd party/cloud host as a \"headless\" Server (no video display or keyboard) on **hardware without audio** running Linux. You can also run a Server in a [**desktop environment**](#servers-on-the-desktop)."
-msgstr ""
+msgstr "De flesta människor kör Jamulus på en tredjeparts/molnvärd som en \"huvudlös\" server (ingen videoskärm eller tangentbord) på **hårdvara utan ljud** som kör Linux. Du kan också köra en server i en [**skrivbordsmiljö**](#servers-on-the-desktop)."
```
Translate text after # in link.

ignotus666 commented on _translator-files/po/sv-SE/Running-a-Server.po:
```diff
@@ -70,126 +71,128 @@ msgid ""
 #, markdown-text, no-wrap
 msgid "**Note** The following steps will result in you running an \"Unregistered\" Server. For more information, see [the Server Modes section](#server-modes) below.\n"
-msgstr ""
+msgstr "**Obs** Följande steg kommer att resultera i att du kör en \"Oregistrerad\" server. För mer information, se [avsnittet Serverlägen](#server-modes) nedan.\n"
```
Translate text after # in link.

ignotus666 commented on _translator-files/po/sv-SE/Running-a-Server.po:
```diff
@@ -244,12 +256,15 @@ msgid ""
 #, markdown-text
 msgid "Add your desired [command line options](Running-a-Server#configuration-options) to the `ExecStart` line in the systemd service file:"
-msgstr ""
+msgstr "Lägg till önskade [kommandoradsalternativ](Running-a-Server#configuration-options) till `ExecStart`-raden i systemd-tjänstfilen:"
```
Translate text after # in link.

ignotus666 commented on _translator-files/po/sv-SE/Running-a-Server.po:
```diff
@@ -352,72 +376,75 @@ msgid ""
 #, markdown-text
 msgid "Any settings made using the graphical interface will be stored in the `Jamulusserver.ini` file. (Do **not** edit this file by hand!) However, some options are not available in the GUI and need to be set using the command line. For more information, see [Servers on the desktop](#servers-on-the-desktop)."
-msgstr ""
+msgstr "Alla inställningar som görs med det grafiska gränssnittet kommer att lagras i filen `Jamulusserver.ini`. (Redigera **inte** den här filen för hand!) Vissa alternativ är dock inte tillgängliga i GUI och måste ställas in med hjälp av kommandoraden. För mer information, se [Server på skrivbordet](#servers-on-the-desktop)."
```
Translate text after # in link.

ignotus666 commented on _translator-files/po/sv-SE/Running-a-Server.po:
```diff
@@ -427,54 +454,57 @@ msgid ""
 #, markdown-text, no-wrap
 msgid "**Note**: Semicolon and newline characters are not allowed in `[name]` and `[city]` within the `--serverinfo` argument. See the [Server mode-related options](#server-mode-related-options) for more information on the country code. \n"
-msgstr ""
+msgstr "**Obs**: Semikolon och nyradstecken är inte tillåtna i `[namn]` och `[stad]` i argumentet `--serverinfo`. Se [Serverrelaterade alternativ](#server-mode-related-options) för mer information om landskoden. \n"
```
Translate text after # in link.

ignotus666 commented on _translator-files/po/sv-SE/Running-a-Server.po:
```diff
@@ -427,54 +454,57 @@ msgid ""
 #, markdown-text
 msgid "Required for a [Registered Server](Running-a-Server#registered-mode). Also required to run Jamulus as a [Directory](Directories)."
-msgstr ""
+msgstr "Krävs för en [Registered Server](Running-a-Server#registered-mode). Krävs också för att köra Jamulus som en [katalog](Directories)."
```
Translate text after # in link.

ignotus666 commented on _translator-files/po/sv-SE/Server-Bandwidth.po:
```diff
@@ -1,67 +1,68 @@
 #, markdown-text
 msgid "{% include breadcrumb.html root=\"More\" branch1=\"Server Administration\" branch1-url=\"Running-a-Server\" %}"
-msgstr ""
+msgstr "{% include breadcrumb.html root=\"Mer\" branch1=\"Serveradministration\" branch1-url=\"Köra-en-server\" %}"
```
Don't translate `branch1-url=\"Running-a-Server\"`

ignotus666 commented on _translator-files/po/sv-SE/Server-Troubleshooting.po:
```diff
@@ -1,68 +1,69 @@
 #, markdown-text
 msgid "{% include breadcrumb.html root=\"More\" branch1=\"Server Administration\" branch1-url=\"Running-a-Server\" %}"
-msgstr ""
+msgstr "{% include breadcrumb.html root=\"Mer\" branch1=\"Serveradministration\" branch1-url=\"Köra-en-server\" %}"
```
Don't translate `branch1-url=\"Running-a-Server\"`

ignotus666 commented on _translator-files/po/sv-SE/Server-Troubleshooting.po:
```diff
@@ -71,111 +72,113 @@ msgid ""
 #, markdown-text
 msgid "If you are registered OK (you can [see it here](https://explorer.jamulus.io/)) and you or your friends can't see your Server, you may need to wait, or start your Client with the `--showallservers` option and try connecting from there ([see command line options](Software-Manual#command-line-options)) on how to start your Client with a config option). In some network configurations, you may  ...
-msgstr ""
+msgstr "Om du är korrekt registrerad (du kan [se det här](https://explorer.jamulus.io/)) och du eller dina vänner inte kan se din server, kan du behöva vänta eller starta din klient med alternativet `--showallservers` och försöka ansluta därifrån ([se kommandoradsalternativ](Software-Manual#command-line-options) för hur du startar din klient med en konfigurationsoption). I vissa nätverkskonfigura ...
```
Translate text after # in link.

ignotus666 commented on _translator-files/po/sv-SE/Server-Troubleshooting.po:
```diff
@@ -71,111 +72,113 @@ msgid ""
 #, markdown-text
 msgid "This should be your **public** IP address (find that with [Google](https://www.google.com/search?q=whatsmyip)). Connect your **own** Jamulus Client to the **local** IP address of your Server (`localhost` or `127.0.0.1` if it's on the same machine as your Client). Note that your public IP address can change - see the note on \"dynamic DNS\" in [this guide](Unregistered-Servers#dynamic-dns-a ...
-msgstr ""
+msgstr "Detta bör vara din **offentliga** IP-adress (hitta den med [Google](https://www.google.com/search?q=whatsmyip)). Anslut din **egen** Jamulus-klient till den **lokala** IP-adressen för din server (`localhost` eller `127.0.0.1` om den är på samma maskin som din klient). Observera att din offentliga IP-adress kan ändras - se noteringen om \"dynamisk DNS\" i [denna guide](Unregistered-Servers ...
```
Translate text after # in link.

ignotus666 commented on _translator-files/po/sv-SE/Server-Troubleshooting.po:
```diff
@@ -71,111 +72,113 @@ msgid ""
 #, markdown-text
 msgid "First of all, make sure you've enabled [port forwarding](Unregistered-Servers#port-forwarding). If you still don't see your server from outside of your local network, you may have issues with your Internet Service Provider (ISP)."
-msgstr ""
+msgstr "Först och främst, se till att du har aktiverat [portvidarebefordran](Unregistered-Servers#port-forwarding). Om du fortfarande inte kan se din server utanför ditt lokala nätverk kan det finnas problem med din internetleverantör (ISP)."
```
Translate text after # in link.

ignotus666 commented on _translator-files/po/sv-SE/Software-Manual.po:
```diff
@@ -98,426 +101,430 @@ msgid ""
 #, markdown-text
 msgid "Note that muting your **own** channel only means you will not hear your signal from the server (and is not advised as it can lead to you becoming out of time with other players). This is therefore not the same as using \"[Mute Myself](#mute-myself-button)\"."
-msgstr ""
+msgstr "Observera att om du tystar din **egen** kanal bara betyder det att du inte kommer att höra din signal från servern (och det rekommenderas inte eftersom det kan leda till att du blir otidsenlig med andra spelare). Detta är därför inte samma sak som att använda \"[Mute Myself](#mute-myself-button)\"."
```
Translate text after # in link.

ann0see replied:
And probably also Mute Myself.

ignotus666 commented on _translator-files/po/sv-SE/Unregistered-Servers.po:
```diff
@@ -1,157 +1,158 @@
 #, markdown-text
 msgid "{% include breadcrumb.html root=\"More\" branch1=\"Server Administration\" branch1-url=\"Running-a-Server\" %}"
-msgstr ""
+msgstr "{% include breadcrumb.html root=\"Mer\" branch1=\"Serveradministration\" branch1-url=\"Köra-en-server\" %}"
```
Don't translate `branch1-url=\"Running-a-Server\"`

pljones commented:
> Translate text after `#` in link.
> Don't translate `branch1-url=\"Running-a-Server\"`

Is there any way these rules can be automated (i.e. based on the generated target links)?  It feels a bit awkward that the id used isn't an absolute value in the source, even if it's translated in the output.  I think both the generated page and id names pretty much have to be guessed at anyway, which is cumbersome.  (Not something for solving here, of course...)

henkdegroot commented:
This fixes #1015

ann0see commented:
@pljones I think we could introduce something in the CI. I don't know if weblate is that feature rich.

ann0see commented:
@ignotus666 I'll look through the content and then - if you think it's good enough for a beta state, we should merge this. I believe we'll see look for and see broken links again after the merge. Is that ok?

ann0see commented on _translator-files/po/es/Software-Manual.po:
```diff
@@ -251,27 +249,23 @@ msgstr "<figure><img src=\"{% include img/es-screenshots/connection-setup-window
 #: ../wiki/en/Software-Manual.md:92
 msgid "Server operators register their servers with Directories. These are either genre-specific or for any genre, or additional [custom entries](#custom-directories). Use the Directory dropdown to select a genre.  You can filter the list by server name or location. To list only occupied Servers, enter a \"#\" character.  If you choose \"Show All Musicians\" the current occupants will be shown."
-msgstr ""
+msgstr "Los operadores de servidores registran sus servidores en Directorios. Éstos pueden ser para un género específico o para cualquiera, o ser [personalizados](#directorios personalizados). Utiliza el menú desplegable de Directorios para seleccionar un género.  Puedes filtrar la lista por nombre de servidor o por ubicación. Para mostrar solo los Servidores ocupados, introduce el caracter \"#\" ...
```
Probably #directorios personalizados will be a broken link. It'll probably be "#directorios-personalizados"

ignotus666 replied:
Yes, you're right.

ignotus666 replied:
Corrected.

ann0see commented on _translator-files/po/it/1-index.po:
```diff
@@ -132,19 +117,19 @@ msgstr "Ti va di collaborare?"
 #, no-wrap
 msgid "_For detailed information on how Jamulus works, see [this paper by Volker Fischer (PDF)](/PerformingBandRehearsalsontheInternetWithJamulus.pdf)._\n"
-msgstr ""
+msgstr "Per informazioni dettagliate su come funziona Jamulus, vedere [questo documento di Volker Fischer PDF](/PerformingBandRehearsalsontheInternetWithJamulus.pdf_\n"
```
```suggestion
msgstr "Per informazioni dettagliate su come funziona Jamulus, vedere [questo documento di Volker Fischer PDF](/PerformingBandRehearsalsontheInternetWithJamulus.pdf)_\n"
```

ann0see commented on _translator-files/po/nl/Running-a-Server.po:
```diff
@@ -882,10 +871,8 @@ msgstr "`sudo systemctl start jamulusTogglerec`"
-#| msgid "You can see the result of these commands if you run `systemctl status jamulus`, or by viewing the logs."
 msgid "You can see the result of these commands if you run `systemctl status jamulus-headless` (or the respective service name you specified manually). You can also view your (sys)log."
-msgstr "Je kunt het resultaat van deze commando's zien als je `systemctl status jamulus` uitvoert of door de logs te bekijken."
+msgstr "Je kunt het resultaat van deze commando's zien als je `systemctl status jamulus-headless`(of de betreffende servicenaam die u handmatig hebt opgegeven) uitvoert. Je kunt ook de (sys)log bekijken."
```
@henkdegroot spacing?

henkdegroot replied:
You've got eagle eyes, corrected on weblate

ann0see commented on _translator-files/po/sv-SE/Getting-Started.po:
```diff
@@ -200,30 +215,34 @@ msgid ""
 "  <figcaption>Connect to a Server</figcaption>\n"
 "</figure>\n"
 msgstr ""
+"<figure>\n"
```
alt text may also need translation.

ann0see commented on _translator-files/po/sv-SE/Include-Backing-Up.po:
```diff
@@ -1,70 +1,70 @@
 #. type: Plain text
 #: ../wiki/en/Include-Backing-Up.md:2
 #, markdown-text, no-wrap
 msgid "<!-- NOTE: This must apply to both Client and Server, and all operating systems -->\n"
```
Seems a bit odd to have a HTML comment translated.

ann0see commented on _translator-files/po/zh-CN/Unregistered-Servers.po:
```diff
@@ -1,146 +1,147 @@
 #: ../wiki/en/Unregistered-Servers.md:28
 msgid "To allow others to connect to your Server from the internet, give them your public IP address. You can [learn your current public IP address using Google](https://www.google.com/search?q=what+is+my+ip).  You should connect yourself using the local network address of the computer the Server is running on. If you are running a Client on the same computer as your Server, the address is `local ...
-msgstr ""
+msgstr "要允许其他人从互联网连接到您的服务器，请向他们提供您的公共 IP 地址。您可以[使用 Google 了解您当前的公共 IP 地址](https://www.google.com/search?q=what+is+my+ip)。您应该使用运行服务器的计算机的本地网络地址连接自己。如果您在与服务器相同的计算机上运行客户端，则地址为“localhost”或“127.0.0.1”。"
```
Maybe it would be better to link some website which is guaranteed available in mainland china. I don't think google is.

ann0see commented on _translator-files/po/sv-SE/Include-Shared-Commands.po:
```diff
@@ -1,88 +1,88 @@
 #. type: Bullet: '- '
 #: ../wiki/en/Include-Shared-Commands.md:14
 #, markdown-text
 msgid "`-i` or `--inifile` Set location of initialization file (overrides default. On macOS simply provide a filename only, since config files can only be read from `/Users/<username>/Library/Containers/io.jamulus.Jamulus/Data/`. For the server replace `io.jamulus.Jamulus` with `io.jamulus.JamulusServer`. Turn on \"Show Library folder\" in \"Show view options\" in Finder to see this folder.)"
```
Source to be updated

ann0see commented on _translator-files/po/sv-SE/Installation-for-Windows.po:
```diff
@@ -228,51 +234,55 @@ msgid ""
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:82
 #, markdown-text
 msgid "Official and further information about how to configure ASIO4ALL is documented in the official [ASIO4ALL FAQs on the ASIO4ALL website](https://www.asio4all.org/index.php/help/faq/){: target=\"_blank\" rel=\"noopener noreferrer\"}."
```
Would be worth updating the link https://www.asio4all.org/index.php/help/faq/ to https://asio4all.org/help/frequently-asked-questions/

ann0see commented on _translator-files/po/sv-SE/Software-Manual.po:
```diff
@@ -98,426 +101,430 @@ msgid ""
 #, markdown-text
 msgid "Opens a dialogue where you can select a server to connect to. If you are connected, pressing this button will end the session.  You can also open the dialogue using \"File&gt;Connection Setup...\". This will leave you connected to any current session until you join another."
-msgstr ""
+msgstr "Öppnar en dialogruta där du kan välja en server att ansluta till. Om du är ansluten avslutas sessionen genom att trycka på den här knappen. Du kan också öppna dialogen med \"File&gt;Connection Setup...\". Detta kommer att lämna dig ansluten till en aktuell session tills du går med i en annan."
```
File>Connection Setup... should probably be translated and adapted to the app.

ann0see reviewed (commented):
Unfortunately still some stuff outstanding.

ann0see commented:
Please re-review your translations for obvious mistakes. I've very high level scanned over some translations, but will continue tomorrow.

ann0see commented on _translator-files/po/sv-SE/Tips-Tricks-More.po:
```diff
@@ -213,108 +216,131 @@ msgid ""
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:98
 #, markdown-text
 msgid "Make sure you connect your MIDI device's output port to the Jamulus MIDI in port (QjackCtl (Linux/Windows), MIDI Studio (macOS) or whatever you use for managing connections). In Linux you will need to install and launch a2jmidid so your device shows up in the MIDI tab in Qjackctl."
```
Source may need to be changed for consistent use of QjackCtl

ann0see commented on _translator-files/po/sv-SE/Tips-Tricks-More.po:
```diff
@@ -213,108 +216,131 @@ msgid ""
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:114
 #, markdown-text
 msgid "With the `-m` command line argument, Server statistics can be generated to be put on a web page."
```
@pljones I think it could be worth discussing deprecating this and integrating it into JSON RPC (not here).

ann0see commented on _translator-files/po/sv-SE/Unregistered-Servers.po:
```diff
@@ -1,157 +1,159 @@
 #. type: Plain text
 #: ../wiki/en/Unregistered-Servers.md:28
 #, markdown-text
 msgid "To allow others to connect to your Server from the internet, give them your public IP address. You can [learn your current public IP address using Google](https://www.google.com/search?q=what+is+my+ip).  You should connect yourself using the local network address of the computer the Server is running on. If you are running a Client on the same computer as your Server, the address is `local ...
```
May also be updated documenting IPv6 support (use ::1) in future

ann0see commented on _translator-files/po/zh-CN/Client-Troubleshooting.po:
```diff
@@ -271,49 +260,48 @@ msgstr "在某些情况下，可能是您的 ISP 阻止了您使用 Jamulus。
 
 #. type: Bullet: '1. '
 #: ../wiki/en/Client-Troubleshooting.md:91
 msgid "Double-click on Jamulus and wait for the above-mentioned message"
```
This will need to be updated. The upcoming macOS version is more involved.

ann0see commented on _translator-files/po/zh-CN/Installation-for-Linux.po:
```diff
@@ -278,15 +265,13 @@ msgstr "启动 Jamulus"
 #: ../wiki/en/Installation-for-Linux.md:94
 msgid "If you have any problems, first check [the troubleshooting page](Client-Troubleshooting)."
-msgstr ""
+msgstr "如果您遇到任何问题，请首先检查[故障排除页面](Client-Troubleshooting)。"
```
Spacing?

ann0see commented on _translator-files/po/zh-CN/Running-a-Server.po:
```diff
@@ -398,137 +374,111 @@ msgstr "如果您想运行多个服务器，也可能在防火墙后面或 LAN 
 #: ../wiki/en/Running-a-Server.md:153
 msgid "You can run a headless Server on Linux using systemd (with `systemctl`), which is covered in other sections.  You can also run a Server directly from the command line. Enter the following command to [run a Registered Server](#server-modes):"
-msgstr ""
+msgstr "您可以在 Linux 上使用 systemd (通过 `systemct`) 运行无头服务器，这在其他部分有所介绍。您也可以直接从命令行运行服务器。输入以下命令以[运行注册服务器](#服务器模式)："
```
systemctl

ann0see commented on _translator-files/po/zh-CN/Running-a-Server.po:
```diff
@@ -939,91 +869,67 @@ msgstr "现在您可以使用 `systemctl` 命令运行这些命令，例如:"
 #: ../wiki/en/Running-a-Server.md:328
 msgid "You can add metadata to the welcome message of a Server to add additional, hidden information like contact information or policy requests for bots (this is comparable to [robots.txt](https://en.wikipedia.org/wiki/Robots.txt)). See the [Community Knowledge Base entry about metadata](/kb/2023/07/30/Server-Metadata.html) for more information."
-msgstr ""
+msgstr "您可以向服务器的欢迎消息中添加元数据，以添加额外的隐藏信息，如联系信息或对机器人的策略请求(这类似于 [robots.txt(https://en.wikipedia.org/wiki/Robots.txt)) (英文维基百科)。有关更多信息，请参阅 [关于元数据的社区知识库条目](/kb/2023/07/30/Server-Metadata.html)。"
```
Missing ]

ann0see reviewed (changes requested):
Still some open stuff.

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS

weblate committed:
Update Italian translation using Weblate

Currently translated at 100.0% (13 of 13 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/jamulus/1-index/it/
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS
Translation: Jamulus/Startpage

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS

weblate committed:
Update Swedish translation using Weblate

Currently translated at 100.0% (16 of 16 strings)

Update Swedish translation using Weblate

Currently translated at 100.0% (27 of 27 strings)

Update Swedish translation using Weblate

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Swedish)

Currently translated at 100.0% (31 of 31 strings)

Update Swedish translation using Weblate

Currently translated at 100.0% (56 of 56 strings)

Update Swedish translation using Weblate

Currently translated at 100.0% (137 of 137 strings)

Update Swedish translation using Weblate

Currently translated at 100.0% (159 of 159 strings)

Update Swedish translation using Weblate

Currently translated at 100.0% (137 of 137 strings)

Update Swedish translation using Weblate

Currently translated at 100.0% (159 of 159 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>
Co-authored-by: ignotus <ignotus666@users.noreply.hosted.weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/client-troubleshooting/sv/
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/sv/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/sv/
Translate-URL: https://hosted.weblate.org/projects/jamulus/server-bandwidth/sv/
Translate-URL: https://hosted.weblate.org/projects/jamulus/server-troubleshooting/sv/
Translate-URL: https://hosted.weblate.org/projects/jamulus/software-manual/sv/
Translate-URL: https://hosted.weblate.org/projects/jamulus/unregistered-servers/sv/
Translation: Jamulus/Client-Troubleshooting
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS
Translation: Jamulus/Running-a-Server
Translation: Jamulus/Server-Bandwidth
Translation: Jamulus/Server-Troubleshooting
Translation: Jamulus/Software-Manual
Translation: Jamulus/Unregistered-Servers

weblate committed:
Update Spanish translation using Weblate

Currently translated at 100.0% (137 of 137 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: ignotus <ignotus666@users.noreply.hosted.weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translate-URL: https://hosted.weblate.org/projects/jamulus/software-manual/es/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS
Translation: Jamulus/Software-Manual

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS

weblate committed:
Update Chinese (Simplified) translation using Weblate

Currently translated at 100.0% (159 of 159 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/zh_Hans/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS
Translation: Jamulus/Running-a-Server

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS

weblate committed:
Update Dutch translation using Weblate

Currently translated at 100.0% (159 of 159 strings)

Translated using Weblate (Dutch)

Currently translated at 100.0% (31 of 31 strings)

Update Dutch translation using Weblate

Currently translated at 100.0% (16 of 16 strings)

Update Dutch translation using Weblate

Currently translated at 100.0% (27 of 27 strings)

Update Dutch translation using Weblate

Currently translated at 100.0% (12 of 12 strings)

Update Dutch translation using Weblate

Currently translated at 100.0% (159 of 159 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Henk De Groot <weblate@hdegroot.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/nl/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/nl/
Translate-URL: https://hosted.weblate.org/projects/jamulus/server-bandwidth/nl/
Translate-URL: https://hosted.weblate.org/projects/jamulus/server-troubleshooting/nl/
Translate-URL: https://hosted.weblate.org/projects/jamulus/unregistered-servers/nl/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS
Translation: Jamulus/Running-a-Server
Translation: Jamulus/Server-Bandwidth
Translation: Jamulus/Server-Troubleshooting
Translation: Jamulus/Unregistered-Servers

ann0see reviewed (commented):
@pljones @ignotus666 I believe the most important issues are resolved now. I'd open a few issues on for changes in the next days but please check if there's still something really pressing before we can merge this.

ignotus666 commented:
I'm away until Monday so can't give it another review, but beyond the stuff already highlighted I think it's ok.

pljones commented:
Do we have enough context just to go in and fix on Weblate the outstanding issues above?

ann0see commented:
Not on every comment. Especially the `#` issues must happen locally on a personal pc anyway

pljones commented:
> Not on every comment. Especially the `#` issues must happen locally on a personal pc anyway

Can the `_site` tarball just be run and switched to the language with the problem and the target id identified that way?

ann0see commented:
Yes. I'd just use the linkchecker we link to find issues

pljones commented:
> Yes. I'd just use the linkchecker we link to find issues

http://jamuluswebsite.drealm.info/wiki/Software-Manual

I think the link checker just spots the breaks, without suggesting fixes?

ann0see commented:
I think we should merge this. It's a pain fixing all this on weblate.

ann0see reviewed (approved):
Approving on the basis that we fix the outstanding broken links soon

pljones commented:
http://jamuluswebsite.drealm.info/wiki/Software-Manual updated with the merge.

ann0see commented:
@genesisproject2020 could you please check for the wrong link targets in your translation of the website as shown above?
